Kanban/Parse/KBNTaskParseAPIManager.h

```diff
@@ -13,7 +13,7 @@
 
 @property KBNParseRequestOperationManager* afManager;
 
-- (void)createTaskWithName:(NSString*)name taskDescription:(NSString*)taskDescription order:(NSNumber*)order projectId:(NSString*)projectId taskListId:(NSString*)taskListId completionBlock:(KBNSuccessDictionaryBlock)onCompletion errorBlock:(KBNErrorBlock)onError;
+- (void)createTaskWithName:(NSString*)name taskDescription:(NSString*)taskDescription order:(NSNumber*)order projectId:(NSString*)projectId taskListId:(NSString*)taskListId priotity:(NSNumber *)priority completionBlock:(KBNSuccessDictionaryBlock)onCompletion errorBlock:(KBNErrorBlock)onError;
```

Comment:
it should be "priority" not "priotity". Same everywhere in the rest of the code.